azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/LoadBalancer.java

```diff
@@ -582,6 +651,8 @@
         UpdateStages.WithBackend,
         UpdateStages.WithLoadBalancingRule,
         UpdateStages.WithInternetFrontend,
+        UpdateStages.WithInternalFrontend,
+        UpdateStages.WithNetworkSubnet,
         UpdateStages.WithInboundNatRule,
         UpdateStages.WithInboundNatPool {
     }
```

Comment:
Looking at LoadBalancerInner `frontend configurations`: it's a collection of `frontend configuration` and each of them can have a reference to ONE public ip address. 

```json
{
  frontendIPConfigurations: [
    {
      "id": frontend-id,
      publicIpAddress: {
        "id": pip-id
      }
    },
    {
      "id": frontend-id,
      publicIpAddress: {
        "id": pip-id
      }
    }
    ...
  ]
}
```

But i see `WithPublicIpAddress<ReturnT>::withExistingPublicIpAddresses(PublicIpAddress...publicIpAddresses)` method here https://github.com/martinsawicki/azure-sdk-for-java/blob/b91e2284e1b474450395671102aa27b940e2e475/azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/LoadBalancer.java#L254 and comment says `This will create a new default frontend for the load balancer under the name "default".` but how can we add multiple of public ips to one frontend ?

Am i missing something?